cli/ballerina-cli/src/test/resources/test-resources/command-outputs/unix/build-bal-project-with-build-tool-not-found.txt

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
A new line is included with "Executing Build Tools" to have a separation between the logs that come before tool execution to make the logs more clear for the reader. For this specific case, it starts with this new line because there are no other outputs before this.